tests/test_utils.py

```diff
@@ -58,8 +58,9 @@ def test_generate_filter_parameters():
     assert expected28 == actual28
 
 
+@pytest.mark.skip(reason="Handled correctly in python 3")
 def test_write_csv_valid_unicode(generate_data, tmpdir):
-    test_data = [u'∆', u'∫', u'∬']
+    test_data = ['∆', '∫', '∬']
```

Comment:
i think we should leave this in place, with the `u'`s, to ensure we still handle unicode correctly in python 2.7

Reply:
also, I think we should use `[u'\u2206', u'\u222b', u'\u222c']` instead of using literal unicode characters in the file, but that's not really part of the python 3 work.